Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "hms-mqtt-publish"
-version = "0.1.0"
+version = "0.2.0"
```

Comment:
minor nitpick. let's leave this at 0.1.0 for now. The version will be increased as part of the release process.